hyprctl/CMakeLists.txt

```
@@ -5,4 +5,17 @@ project(
     DESCRIPTION "Control utility for Hyprland"
 )
 
-add_executable(hyprctl "main.cpp")
\ No newline at end of file
+add_executable(hyprctl "main.cpp")
+
+# binary
+install(TARGETS hyprctl)
+
+# shell completions
+install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/hyprctl.bash
+        DESTINATION ${CMAKE_INSTALL_DATADIR}/bash-completion/completions
+        RENAME hyprctl)
+install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/hyprctl.fish
+        DESTINATION ${CMAKE_INSTALL_DATADIR}/fish/vendor_completions.d)
+install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/hyprctl.zsh
+        DESTINATION ${CMAKE_INSTALL_DATADIR}/zsh/site_functions
```

Typo? s/_/-/ to match Makefile (and Meson):
```suggestion
        DESTINATION ${CMAKE_INSTALL_DATADIR}/zsh/site-functions
```